File: Week-07/day-01/12 second-in-a-day/seconds_in_a_day.js

'use strict';



var currentHours = 14;

var currentMinutes = 34;

var currentSeconds = 42;


console.log("Time remaining is: " + ((24 - currentHours - 1) * 3600 + (60 - currentMinutes - 1) * 60 + (60 - currentSeconds))); 
// Write a program that prints the remaining seconds (as an integer) from a

// day if the current time is represented by these variables

File: Week-02/day-01/seconds_in_a_day.py
current_hours = 14
current_minutes = 34
current_seconds = 42

# Write a program that prints the remaining seconds (as an integer) from a
# day if the current time is represented bt the variables
secondsaday = 60 * 60 * 24

currenthour = 14 * 60 * 60
currentminute = 34 * 60

currenttime = currenthour + currentminute + current_seconds
remaining = (secondsaday - currenttime)
print(remaining)
